tests/AppTest.php

```diff
@@ -68,4 +70,22 @@ public function testUnexpectedOutputLateError(): void
             ob_end_clean();
         }
     }
+
+    /* throws headers already sent exception so not sure how to write this test for stream output
+    public function testStreamResponse(): void
```

Comment:
this test should test sending large (10GB+) data with known pattern to test the response is not materialized somewhere

Reply:
To be honest I have no idea how to write that. Will have to ask ChatGPT :)

Reply:
implemented - DemosTest tested with 60GB, DemosHttpTest seems to buffer the output on disk, not exactly sure why, probably due some php cli server/guzzle buffering - I will look shortly tomorrow, otherwise the PR is complete

Reply:
it seems PHP cannot flush data faster than about 150 MB/s, I have opened a php-src issue for it

Reply:
the main bottlenect seems to be in Guzzle HTTP client - https://github.com/guzzle/guzzle/issues/3115

until improved, let's test the streaming support with 50 MB only